llpc/test/shaderdb/ShaderWFImageSample.spvasm

```diff
@@ -0,0 +1,74 @@
+; BEGIN_SHADERTEST
```

Comment:
Is it possible to rename it to OpTypeSampledImage_TestWaterfallInsertion.spvasm to follow existing test naming? Also, if such case could be achieved by a GLSL source, that is better. This is because SPIR-V assembly is fragile to any spec upgrading, which could break the testing once spvgen is updated by incorporating new SPIR-V validation sources.

Reply:
This has already been merged - however, there's another change #1011 that adds an extra check to the test. I'll rename it there (and also take a look at redoing it as a glsl source test instead).

Reply:
Thanks. That's OK.